packages/fxa-auth-server/scripts/test-ci.sh

```diff
@@ -17,6 +17,7 @@ node -r esbuild-register ./scripts/oauth_gen_keys.js
 node ../db-migrations/bin/patcher.mjs
 
 yarn run merge-ftl:test
+yarn run emails-scss
```

Comment:
@vbudhram I'm not really sure why I have to add this here. I tried adding `yarn emails-scss` to the `test-ci` command, as well as others because IIRC correctly there were failures in "test local" without this, and through all my trial and errors of pushing, this is what worked.